smogn/over_sampling.py

```diff
@@ -12,12 +12,12 @@
 def over_sampling(
     
     ## arguments / inputs
-    data,       ## training set
-    index,      ## index of input data
-    perc,       ## over / under sampling
-    pert,       ## perturbation / noise percentage
-    k           ## num of neighs for over-sampling
-    
+    data,               ## training set
+    index,              ## index of input data
+    perc,               ## over / under sampling
+    pert,               ## perturbation / noise percentage
+    k,                  ## num of neighs for over-sampling
+    random_seed = None  ## random seed for sampling (pos int or None)
```

Comment:
@ArnoldYSYeung are you open to changing the `random_seed` argument to simply `seed` for brevity?